frappe/public/js/frappe/form/controls/text_editor.js

```diff
@@ -203,91 +199,9 @@ frappe.ui.form.ControlTextEditor = frappe.ui.form.ControlCode.extend({
 
 	get_input_value() {
 		let value = this.quill ? this.quill.root.innerHTML : '';
-		// quill keeps ol as a common container for both type of lists
-		// and uses css for appearances, this is not semantic
-		// so we convert ol to ul if it is unordered
-		const $value = $(`<div>${value}</div>`);
-		$value.find('ol li[data-list=bullet]:first-child').each((i, li) => {
-			let $li = $(li);
-			let $parent = $li.parent();
-			let $children = $parent.children();
-			let $ul = $('<ul>').append($children);
-			$parent.replaceWith($ul);
-		});
-		value = this.convertLists($value.html());
```

Comment:
@surajshetty3416 any reason why this was removed? Seems like the problem that was addressed here still exists.